python/snewpy/neutrino.py

```diff
@@ -91,4 +124,109 @@ def get_mixing_angles(self):
             Angles theta12, theta13, theta23.
         """
         return (self.theta12, self.theta13, self.theta23)
+        
+    def get_mass_squared_differences(self):
+        """Mass squared differences .
+        
+        Returns
+        -------
+        tuple
+            dm21_2, dm31_2, dm32_2.
+        """        
+        return (self.dm21_2, self.dm31_2, self.dm32_2)
+
+@dataclass
+class MixingParameters4Flavor(MixingParameters3Flavor):
+    """A class for four flavor neutrino mixing. 
+    ..Note: it is an extension of :class:`MixingParameters3Flavor`, and can be constructed using it:
+    
+        >>> pars_3f = MixingParameters() #standard 3flavor mixing
+        >>> pars_4f = MixingParameters4Flavor(**pars_3f, theta14=90<<u.deg, dm41_2=1<<u.GeV**2)
+    """
+    #sterile neutrino miging angles
+    theta14: u.Quantity[u.deg] = 0<<u.deg
+    theta24: u.Quantity[u.deg] = 0<<u.deg
+    theta34: u.Quantity[u.deg] = 0<<u.deg
+    #sterile neutrino mass squared differences
```

Comment:
why setting them to zero instead of being an input parameter the user can give?

Reply:
Note that this is marked as a [dataclass](https://docs.python.org/3.11/library/dataclasses.html), which automatically generates a `__init__` function to set them as input parameters. The value defined here is used as the default.

Reply:
Yes, as Jost said zeros will be default parameters in the constructor. So by default there is no mixing with sterile neutrinos, and only if user provides the mixing angles in the constructor, e.g.:
```python
params4f = MixingParameters4Flavor(**params3f, theta24=45<<u.deg)
```
there will be an effect of sterile neutrinos